baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -474,8 +476,17 @@ def create_bm_configs(self, req):
         rsp = AgentResponse()
 
         # check preconfiguration md5sum
+        # py2
         if hashlib.md5(cmd.preconfigurationContent).hexdigest() != cmd.preconfigurationMd5sum:
```

Ruff (0.15.6)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)